lib/pipeline/webpackProcess.js

```diff
@@ -1,208 +1,114 @@
-const async = require('async');
-const webpack = require('webpack');
-const utils = require('../utils/utils');
-const fs = require('../core/fs');
 const constants = require('../constants');
-const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
+const fs = require('../core/fs');
 const ProcessWrapper = require('../core/processes/processWrapper');
-const path = require('path');
-const glob = require('glob');
+const webpack = require('webpack');
+const writeFile = require('util').promisify(require('fs').writeFile);
 
 let webpackProcess;
 
+function errorMessage(e) {
```

Comment:
I have seen this pattern a lot shall we make that function more global?

Reply:
Sure, maybe in utils; do you think that's the best place?

Reply:
Yes, utils is fine by me

Reply:
Done.